### ISSUE TITLE
Ruby SDK Bug 🐞

### ISSUE THREAD

**Issue by pietrop** [remotion-dev/remotion#4619]
# Bug Report 🐞

Thanks for adding ruby SDK in PR https://github.com/remotion-dev/remotion/pull/4552 🙏 

I installed the ruby gem [remotion_lambda](https://rubygems.org/gems/remotion_lambda) in our rails app [and followed these docs for integration](https://www.remotion.dev/docs/lambda/ruby).
However the installation fails, and I suspect it's because the ruby gem it's not in the right shape.

Looking at the installed files locally it was 
```
.
└── lib
   ├── client.rb
   ├── render_media_on_lambda_payload.rb
   ├── render_progress_payload.rb
   ├── render_still_on_lambda_payload.rb
   ├── s3_output_provider.rb
   ├── sdk.rb
   └── version.rb
```

instead it should be 
```
.
├── lib
│   ├── remotion_lambda
│   │   ├── client.rb
│   │   ├── render_media_on_lambda_payload.rb
│   │   ├── render_progress_payload.rb
│   │   ├── render_still_on_lambda_payload.rb
│   │   ├── s3_output_provider.rb
│   │   ├── sdk.rb
│   │   └── version.rb
│   └── remotion_lambda.rb
└── remotion_lambda.gemspec

3 directories, 9 files
```
And after making these changes locally I got it working ⬇ 

- [ ] The `/remotion_lambda.gemspec` is not included in the gem bundle after install, altho both remotion ruby package](https://github.com/remotion-dev/remotion/tree/main/packages/lambda-ruby) and [separate repo]( https://github.com/remotion-dev/lambda-ruby-sdk) have it.
  - [ ] I don't see anything that stands out in [remotion/packages/lambda-ruby/publish.ts](https://github.com/remotion-dev/remotion/blob/main/packages/lambda-ruby/publish.ts) that would indicate that  `/remotion_lambda.gemspec`  is being omitted.  In fact [according to this line](https://github.com/remotion-dev/remotion/blob/main/packages/lambda-ruby/publish.ts#L25-L29) it should have been copied over. So not sure what is going on.
- [ ] it' s also missing `lib/remotion_lambda.rb` as an entry point for the gem.
```ruby 
require 'aws-sdk-lambda'
require 'json'
require 'logger'
require_relative 'remotion_lambda/version'
require_relative 'remotion_lambda/client'
require_relative 'remotion_lambda/render_media_on_lambda_payload'
require_relative 'remotion_lambda/render_progress_payload'
require_relative 'remotion_lambda/render_still_on_lambda_payload'
require_relative 'remotion_lambda/s3_output_provider'

require_relative 'remotion_lambda/sdk'

module RemotionLambda
  # This module serves as the namespace for the RemotionLambda gem.
  # Add any top-level methods, constants, or configuration here if needed.
end
```
  - [ ] This  `lib/remotion_lambda.rb`  would probl need to be added to [`lib`](https://github.com/remotion-dev/remotion/tree/main/packages/lambda-ruby/lib) and  copied over via [publish.ts](https://github.com/remotion-dev/remotion/blob/main/packages/lambda-ruby/publish.ts)
- [ ] all files in `lib` with the exception of `lib/remotion_lambda.rb` need to be inside of `lib/remotion_lambda`
  - [ ] [could probl be adjusted around this line](https://github.com/remotion-dev/remotion/blob/main/packages/lambda-ruby/publish.ts#L24)

Some docs [make your own gem|guide](https://guides.rubygems.org/make-your-own-gem)

**Comment by JonnyBurger** [comment]
Thanks, I made a more realistic example now where there is a second package that installs the Gem.

From that I can run it now after I fixed the bugs you pointed out!

**Comment by JonnyBurger** [comment]
Fixed by #4620
